addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1424,6 +1424,8 @@ const MediapickerUserValueWidget = UserValueWidget.extend({
             noIcons: true,
             noDocuments: true,
             isForBgVideo: true,
+            vimeoPreviewIds: ['299225971', '414790269', '420192073', '368484050', '334729960', '417478345'
+                '312451183', '415226028', '367762632', '340475898', '374265101', '370467553'],
```

Comment:
There is a typo here... so the branch crashes as soon as the page loads :/ Could you rebase and fix please?